test/client_test.coffee

```diff
@@ -67,11 +67,11 @@ describe 'Basic test for the 3Scale::Client', ->
     it 'should call the callback with a successful response', (done) ->
       nock('https://su1.3scale.net')
         .get('/transactions/authorize.xml')
-        .query({ app_key: 'bar', app_id: 'foo', provider_key: '1234abcd' })
+        .query({ service_id: '1234567890987', app_key: 'bar', app_id: 'foo', provider_key: '1234abcd' })
         .reply(200, '<status><authorized>true</authorized><plan>Basic</plan></status>')
 
       client = new Client '1234abcd'
-      client.authorize {app_key: 'bar', app_id: 'foo'}, (response) ->
+      client.authorize {service_id: '1234567890987', app_key: 'bar', app_id: 'foo'}, (response) ->
```

Comment:
You added the service id in two of the calls of this test file, but there are more. For example, the authorize call below this one (line 82).